flake.nix

```diff
@@ -324,8 +324,22 @@
 
           nixpkgsLibTests =
             forAllSystems (system:
-              import (nixpkgs + "/lib/tests/release.nix")
-                { pkgs = nixpkgsFor.${system}.native;
+              let
+                pkgs = nixpkgsFor.${system}.native;
+                # Kludge until we can merge this: https://github.com/NixOS/nixpkgs/pull/274077
+                nixpkgsPatched = pkgs.applyPatches {
```

Comment:
The `lib` test suite has a few test cases that check the output of `nix-instantiate --eval-only`. I don't think that output is guaranteed by any compatibility policy, so I think we're good to change it, but we do need to patch the tests. Once we have the output finalized here, we can merge this PR in nixpkgs and then update the Flake input reference and remove this patch.